### ISSUE TITLE
Callback on enter request for unauthorized users

### ISSUE THREAD

**Issue by iospro** [jitsi/jitsi-meet#10295]
It would be useful to add callback on enter request for unauthorized users.
For example, in iOS `JitsiMeetViewDelegate` we have callback

`- (void)participantJoined:(NSDictionary *)data`

When unauthorized user comes we have on-screen message with two buttons Allow/Disallow.
It would be nice to have some callback as

`- (void)participantIsWaiting:(NSDictionary *)data`

`data` contains basic information on a new user (some `userGuid`)

Also we need some method to Allow/Disallow to enter in parallel with current in-box mechanism

- (void)enterParticipant:(NSString *)userGuid  allow:(BOOL)allow

This SDK extension allows to notify admin about new waiting participants while screencasting outside the custom Jitsi App.
In such case the App can make a local notification with interactive buttons on `participantIsWaiting` event to allow/disallow enter for new participant without need to enter the Jitsi App

This feature is needed in all platforms.

**Comment by saghul** [comment]
You mean lobby actions, right?

**Comment by iospro** [comment]
Yes, incoming lobby event and programmical reaction on this event via SDK API

**Comment by saghul** [comment]
Got it. I don't think we have short term plans to add this, but any PR is welcome.

**Comment by iospro** [comment]
Could you please describe how we can get to know which commits from master branch were included into some SDK release? We can't find release notes and commit relations in SDK repo

**Comment by saghul** [comment]
It's right here: https://github.com/jitsi/jitsi-meet-release-notes/blob/master/CHANGELOG-MOBILE-SDKS.md

That includes the release notes and links to the tag and list of commits on a given release.

It's also linked in the "releases" tab on the handbook: https://jitsi.github.io/handbook/docs/releases

**Comment by iospro** [comment]
Could you please make a suggestion about the date of the next release iOS/android sdk? We made some pull requests with very important for us localization improvements.

**Comment by saghul** [comment]
We are hoping for a release late this week or the next.

**Comment by iospro** [comment]
very hope this week
